libzmix/src/signatures/bbs/mod.rs

```diff
@@ -0,0 +1,5 @@
+pub mod keys;
+pub mod pok_sig;
+pub mod signature;
+
+// TODO: Add "setup" to generate g1 and g2 and use them in-place of G1::generator() and G2::generator()
```

Comment:
Is there a ticket somewhere describing the work that needs to be done for this TODO?

Reply:
Some issuers might not trust the generation of `G1::generator()` and `G2::generator()`. The verifier should be convinced that the discrete log them wrt. others is not known.